Adding documentation for certificate renewal (#2293)
dapr/docs: daprdocs/content/en/operations/security/mtls.md

4. Restart your Dapr applications to pick up the latest trust bundle.

> {{% alert title="Recommended CLI command to rotate certificates with new ones" color="success" %}}

Reviewer: It is hard to separate out the kubectl approach and the Dapr CLI approach here. Can you add headings for each so that these are separated? For example "Root certificate upgrade using kubectl" and "Root certificate upgrade using CLI"

Reviewer: Also my concern here, is that given the CLI is the preferred method we should have this first BEFORE the kubectl method. So could you switch this around in the order?

Author: I am now keeping these in separate section. Kindly have a look at recent changes.

> The below CLI command can be used to renew root and issuer certificates of your Kubernetes cluster. It also gives an option to provide validity for the new certificates in number of days. 

Reviewer: ```suggestion
The CLI command below can be used to renew root and issuer certificates in your Kubernetes cluster. It also gives an option to provide validity for the new certificates in number of days. 
```

> The 2nd variant of this command provides `--private-key` flag to support use of `private root key` of your existing deployed certificate.

Reviewer: It is hard to separate out the kubectl approach and the Dapr CLI approach here. Can you add headings for each so that these are separated? For example "Custom root certificate upgrade using kubectl" and "Custom root certificate upgrade using CLI"

> The 2nd variant of this command provides `--private-key` flag to support use of `private root key` of your existing deployed certificate.

Reviewer: ```suggestion
The alternative variant of this command provides `--private-key` flag to support the use of a `private root key` for your existing deployed root certificate.
```

> This command generate new certificates signed by same private key. In this case the `Dapr Sentry service` would be able to pick the new certificate without restarting it.

Reviewer: ```suggestion
This command generates new certificates signed by the same private key. In this case the `Dapr Sentry service` can then read these new certificates without restarting.
```

kubectl rollout restart deploy/dapr-operator -n <DAPR_NAMESPACE>
kubectl rollout restart statefulsets/dapr-placement-server -n <DAPR_NAMESPACE>
```

> {{% alert title="Recommended CLI command to achieve above steps" color="success" %}}

Reviewer: It is hard to separate out the kubectl approach and the Dapr CLI approach here. Can you add headings for each so that these are separated? For example "Custom root certificate upgrade using kubectl" and "Custom root certificate upgrade using CLI"

Reviewer: Also my concern here, is that given the CLI is the preferred method we should have this first BEFORE the kubectl method. So could you switch this around in the order?

> To update the provided certificates in Kubernetes cluster, the belowl CLI command can be used.

Reviewer: ```suggestion
To update the provided certificates in the Kubernetes cluster, the CLI command below can be used.
```

*Note: If you signed the cert root with a different private key, restart the Dapr instances.*

> ## Community call video on certificate rotation

Reviewer: I realize this video is lost at the end of this article and is best move to the end of the certificate renewal section. Could you move this video up to the end of this section?
